app/render/index.ts

```diff
@@ -62,19 +66,45 @@ const nightPDF = (async function () {
 	let _hueSliderElement: HTMLElement;
 	let _extraBrightnessSliderElement: HTMLElement;
 	let _splashElement: HTMLElement;
-	let _splashExtraElement: HTMLElement;
 	let _defaultButton: HTMLElement;
 	let _sepiaButton: HTMLElement;
 	let _redeyeButton: HTMLElement;
 	let _customButton: HTMLElement;
+	let _tabGroup: TabGroup | null = null;
+	let _slidersInitialized = false;
+	const _tabCssKeyMap: Map<Tab, string> = new Map();
 
 	async function main() {
 		_appContainerElement = document.getElementById(
 			"appContainer",
 		) as HTMLElement;
-		_pdfElement = document.getElementById("pdfjs") as HTMLIFrameElement;
+		_appContainerElement.style.display = "none";
+		_tabGroup = document.querySelector("tab-group");
+		_tabGroup?.on("ready", (tabGroup: TabGroup) => {
+			// replace new tabe default "click" event handler
+			tabGroup.buttonContainer.getElementsByTagName("button")[0].addEventListener(
```

Comment:
does this have to be hardcoded to the first element? is there a possibility for `getElementsByTagName` to return > 1 elements? what are the ramifications if it does?

Reply:
It's fragile, but the button itself doesn't get an ID or class, and at least from what I see in the source `electron-tabs` only adds a single button element during the tab-group setup: https://github.com/brrd/electron-tabs/blob/49525b8634e949f8875f47dd4700cfe0c94e41a3/src/index.ts#L139

Reply:
as long as upstream is okay with this I am too!